ddocs/medic/views/docs_by_replication_key/map.js

```diff
@@ -1,5 +1,6 @@
 function (doc) {
   if (doc._id === 'resources' ||
+      doc._id === 'branding' ||
```

Comment:
This is the important bit - this view is how we work out how to replicate docs to offline users. Adding this makes sure all users can get it.

Reply:
I see